lib/spack/docs/build_settings.rst

```diff
@@ -179,15 +179,72 @@ Specific limitations include:
 
 * A package must define ``executables`` and ``determine_spec_details``
   for Spack to locate instances of that package.
-* This is currently intended to find build dependencies rather than
-  library packages.
+* This implementation only collects and examines executable files, so it
```

Comment:
Say "the current implementation" (it reads better since this appears in the docs and not in a PR)